packages/matter-node.js/test/IntegrationTest.ts

```diff
@@ -212,6 +226,32 @@ describe("Integration", () => {
         });
     });
 
+    describe("storage", () => {
```

Comment:
Can unit tests for storage be dedicated unit test rather than mixed with Integration?

Reply:
In fact for me the Integration test is more an end-to-end test so I would also include verificstion that the right stuff is stored in there.

I currently provide a separate PR to more add testing for storage

Reply:
I just like leading with functionality getting added as dedicated classes with dedicated tests.

Reply:
I agree, this was missed in the original PR that brought in Persistence ... I will add testing to this PR tomorrow